src/index.ts

```diff
@@ -19,21 +19,42 @@ async function run(): Promise<void> {
     }
 
     const octokit = new Octokit(options);
-    const pullRequest = await octokit.pulls.create(pullParams);
-    const pullNumber = pullRequest.data.number;
-    const htmlUrl = pullRequest.data.html_url;
-
-    if (reviewers.length > 0) {
-      await octokit.pulls.createReviewRequest({
-        owner: pullParams.owner,
-        repo: pullParams.repo,
-        pull_number: pullNumber,
-        reviewers,
-      });
-    }
+    try {
+      const pullRequest = await octokit.pulls.create(pullParams);
+      const pullNumber = pullRequest.data.number;
+      const htmlUrl = pullRequest.data.html_url;
+
+      if (reviewers.length > 0) {
+        await octokit.pulls.createReviewRequest({
+          owner: pullParams.owner,
+          repo: pullParams.repo,
+          pull_number: pullNumber,
+          reviewers
+        });
+      }
+
+      if(labels.length > 0) {
+        await octokit.issues.addLabels({
+          owner: pullParams.owner,
+          repo: pullParams.repo,
+          issue_number: pullNumber,
+          labels: labels,
+        });
+      }
 
-    setOutput('number', pullNumber.toString());
-    setOutput('html_url', htmlUrl);
+      setOutput('number', pullNumber.toString());
+      setOutput('html_url', htmlUrl);
+      setOutput('pr_created', 'true');
```

Comment:
Can we name this one just `created`? We are in the context of a PR here anyway so the naming doesn't need to repeat that.

Reply:
fixed